.github/workflows/workflow-sync.yml

```diff
@@ -47,6 +47,7 @@ env:
     gocodebox/lifterlms-stripe
 
   PHP_FILES: |
+    .github/workflow-templates/.editorconfig=.editorconfig
```

Comment:
what repo am I missing that requires it to be duplicated in both util and php file lists?

Reply:
Ah! Yes, I agree.